### PR TITLE
Feature/azaytsev/g api docs

### PR DESCRIPTION
**Pull request by andrew-zaytsev**

### CODE REVIEW

File: docs/gapi/gapi_intro.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Introduction to OpenCV Graph API (G-API) {#openvino_docs_gapi_gapi_intro}
```
Comment (ilya-lavrenov) [inline docs/gapi/gapi_intro.md:1]: why do we need all this stuff in our docs?  It's OpenCV module and all documentation must be there

Comment (ilya-lavrenov) [reply docs/gapi/gapi_intro.md:1]: CC @dmatveev @alalek

Comment (andrew-zaytsev) [reply docs/gapi/gapi_intro.md:1]: The G-API documentation was integrated by request from Dmitry @dmatveev .

**Review by ntyukaev — APPROVED** [review]

File: docs/gapi/gapi_intro.md (hunk 1 above)
Comment (ilya-lavrenov) [reply docs/gapi/gapi_intro.md:1]: ok, question is redirected to @dmatveev - why do we need this docs inside OpenVINO? Such intro must be a part of OpenCV docs, we can just refer to it, I suppose

Comment (dmatveev) [reply docs/gapi/gapi_intro.md:1]: Customers asked for some starting point at the OpenVINO docs as G-API was exposed and served there under the OpenVINO umbrella.

OpenCV docs have this already & the operation/kernel reference will remain there. The OpenVINO-side part will be more Intel/OpenVINO-specific as
- more *PU support comes in the future;
- more OMZ samples get ported on G-API.

Comparison with core/imgproc is not very correct as those parts are quite stable, well known, and not evolve _that_ fast as we do.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**